packages/runtime-dom/src/apiCustomElement.ts

```diff
@@ -208,7 +210,31 @@ export class VueElement extends BaseClass {
       if (this._resolved) {
         this._update()
       } else {
-        this._resolveDef()
+        let parent: Node | null = this
+        let isParentResolved = true
+        // locate nearest Vue custom element parent and set it to '_ce_parent‘
+        while (
```

Comment:
I am wondering if the `connectedCallback` might be called multiple times with the same instance (within a `v-if` for example) ?
If so I think we might avoid looping again if we check on `this._ce_parent` (unless `this._resolved` would be already true on next triggers of the `connectedCallback`